docs/contents/releases.md

```diff
@@ -1,128 +1,159 @@
+specified in the controller's `go.mod` file.
 
-The GitHub Issue **WILL NOT** be associated with a GitHub Milestone.
+The ACK code generator that produces Go code for service controllers depends on
+a specific version of the ACK common runtime.
 
-## PLANNED
+!!! note "dependency between the code generator and common runtime"
+    The ACK code generator depends on the ACK common runtime in a unique way:
+    the Go code that the ACK code generator *produces* adheres to a specific
+    version of the ACK common runtime. Even though no Go code in the ACK code
+    generator actually imports the ACK common runtime, this dependency exists
+    because the Go code produced by the templates inside the code generator imports
+    the ACK common runtime. In order to make this Go code dependency more strict,
+    we have a test package inside the ACK code generator that imports the ACK
+    common runtime. In this way, we're able to include a version-specific
+    dependency line in the ACK code generator's `go.mod` file, thereby allowing
+    Go's module infrastructure to pin the dependency between the code generator and
+    the common runtime.
 
-The `PLANNED` status indicates that **the AWS service is on our radar** for
-building into ACK.
+## Stable Helm Charts
 
-There will be a Github Issue for tracking the build of the ACK service
-controller for the service.
+We [recommend][recommend-helm] using Helm to install an ACK service controller.
 
-The GitHub Issue **WILL BE** associated with a GitHub Milestone indicating the
-target date for the `DEVELOPER PREVIEW` release of that service's controller.
+[recommend-helm]: https://aws-controllers-k8s.github.io/community/user-docs/install/#helm-recommended
 
-## BUILD
+If you want to only install ACK service controllers that have been determined
+to be stable by the ACK contributor team and the associated AWS service team,
+you may choose to install the Helm chart for an ACK service controller that has
+been tagged with the `-stable` suffix -- e.g. `elasticache-stable`.
 
-The `BUILD` status indicates that the ACK service controller for the AWS
-service is **actively being built** in preparation for `DEVELOPER PREVIEW`
-release of that ACK service controller.
+This `-stable` artifact tag points to a Helm chart that has configuration values that
+have been tested with a specific SemVer Docker image for the ACK service
+controller. Typically these tests are "soak" tests and allow the team
+maintaining that ACK controller's source code to have a high degree of
+confidence in the controller's long-running operation.
 
-It is in the `BUILD` status that we will identify those **AWS service API
-resources** that will be supported in `DEVELOPER PREVIEW` and which will be
-supported by the `GENERALLY AVAILABLE` release of the controller.
+!!! "important"
+    Please note that not all ACK service controllers have Helm chart that have
+    been tagged with the `-stable` suffix.
```

Comment:
If I understood correctly, we will start publishing charts with `-stable` sufix everytime there is a new SemVer (which is, after they hit GA), right? But at that point, will we continue to publish the charts without the `-stable` sufix?

Reply:
I think after first `-stable` release there will be need to have releases that are **not** tagged as '-stable', so that work (development and testing) to support additional resources can be done in preparation to next  '-stable' release.

Reply:
Would there be "latest" tagged images (even after a stable release) that represent non-stable but with most recent changes from 'main' branch?